Week-5/mapcount.js

```diff
@@ -0,0 +1,37 @@
+function vowelCount(text) {
+  let vowelMap = new Map();
+  let i = 0;
+  var temp = text.toString();
+
+  for (i = 0; i < temp.length; i++) {
+    if (
+      temp[i] == "a" ||
+      temp[i] == "e" ||
+      temp[i] == "i" ||
+      temp[i] == "o" ||
+      temp[i] == "u"
+    ) {
+      vowelMap.set(i, temp[i]);
+    } else if (
+      temp[i] == "A" ||
+      temp[i] == "E" ||
+      temp[i] == "I" ||
+      temp[i] == "O" ||
```

Comment:
@rajeshvlega9 Looks good to me. But instead of checking all characters by OR condition. Can you have a single string as 'aeiou' for vowels and check whether each char is part of this

for ex: ['a', 'e', 'i', 'o', 'u'].includes('a') || ['a', 'e', 'i', 'o', 'u'].includes('r'). This will return true or false based on this we can push to the map.

Reply:
But what u did is correct, just a small improvement